mmv1/templates/terraform/examples/scc_mute_config.tf.erb

```diff
@@ -0,0 +1,6 @@
+resource "google_scc_mute_config" "<%= ctx[:primary_resource_id] %>" {
+  mute_config_id = "<%= ctx[:vars]['mute_config_id'] %>"
```

Comment:
you do not need to have a ctx[:vars] label for every variable within the example. Unless they are names/id's, you can just manually write them in the example